### PR TITLE
protocol: add PrefetchProgress to stat/sync status

### PR DESCRIPTION
**Pull request by strib**
And print out progress in the `fs stat` and `fs sync show` commands.

Issue: KBFS-3575

### CODE REVIEW

File: go/client/cmd_simplefs_stat.go [hunk 1]
```diff
@@ -101,7 +113,7 @@ func (c *CmdSimpleFSStat) Run() (err error) {
 				r.Revision, keybase1.FormatTime(e.Time),
 				keybase1.DirentTypeRevMap[e.DirentType],
 				e.Size, e.Name, e.LastWriterUnverified.Username,
-				e.PrefetchStatus)
+				prefetchStatusString(e))
```
Comment (jakob223) [inline go/client/cmd_simplefs_stat.go]: I feel like this output has just gotten less and less human-intelligible. Do we intend for humans to read the output of this command, or just scripts?

File: go/client/cmd_simplefs_stat.go [hunk 2]
```diff
@@ -56,6 +57,17 @@ func NewCmdSimpleFSStat(cl *libcmdline.CommandLine, g *libkb.GlobalContext) cli.
 	}
 }
 
+func prefetchStatusString(e keybase1.Dirent) string {
+	if e.PrefetchStatus != keybase1.PrefetchStatus_IN_PROGRESS ||
```
Comment (jakob223) [inline go/client/cmd_simplefs_stat.go]: I think maybe it's confusing to say "sync in progress" only when there are 0/n bytes synced. Not sure if I have any better alternatives. "Sync waiting to start" or something?

File: go/client/cmd_simplefs_stat.go (hunk 1 above)
Comment (strib) [reply go/client/cmd_simplefs_stat.go]: Yeah this output is a mess, I agree.  But we've never had a proper design for it.  I guess we just need headers or something, but that seems like the job for a future PR, if anyone cares enough to do it.

File: go/client/cmd_simplefs_stat.go (hunk 2 above)
Comment (strib) [reply go/client/cmd_simplefs_stat.go]: This is actually when 0 is the denominator, that is we haven't even fetched a single block yet to look at it.  I'm changing this to `keybase1.PrefetchStatus_NOT_STARTED.String()` in that case.

**Pull request merged**